### PR TITLE
Version 4

### PR DESCRIPTION
**Pull request by thompsnm**
Support Protractor `v4` and Cucumber `v1`

PR for #39

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
-    - "4.1"
-    - "4.0"
+    - "4.4"
+    - "4.3"
 before_install:
     - "export DISPLAY=:99.0"
     - "sh -e /etc/init.d/xvfb start"
+    - "npm install -g npm@3"
```
Comment (thompsnm) [inline .travis.yml]: NVM install `npm@2` with Node `v4` by default.

**Comment by thompsnm** [comment]
I'll let this PR soak for a few days. If no one raises concerns by mid next week I'll move forward with merging and publishing this branch, and deprecating `v1`.

File: Gruntfile.coffee [hunk 2]
```diff
@@ -1,85 +0,0 @@
-module.exports = (grunt) ->
```
Comment (thompsnm) [inline Gruntfile.coffee]: No one is proactively supporting the [`grunt-protractor-runner`](https://www.npmjs.com/package/grunt-protractor-runner) project, so I decided to try ripping out Grunt altogether. The scripts necessary to create the same functionality were not too difficult, so I'm advocating for just managing the build process using `npm`.

**Comment by wyvern8** [comment]
looks good to me.

**Pull request merged**